api/v1beta1/machineset_types.go

```diff
@@ -129,6 +130,30 @@ const (
 	OldestMachineSetDeletePolicy MachineSetDeletePolicy = "Oldest"
 )
 
+// ANCHOR: RemediationStrategy
+
+// RemediationStrategy allows to define how the MachineSet can control scaling operations.
+type RemediationStrategy struct {
```

Comment:
Nit: Should we move this to `machinedeployment_types.go` as the struct is used there?

Reply:
done